### ISSUE TITLE
"Docs" link is not visible in the "Visual studio code page":A11y_VisualStudioCodeServices_Overview_Infoandrelationships

### ISSUE THREAD

**Issue by PrasannaPolisetty123** [microsoft/vscode-docs#7408]
**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver".**
## GitHub Tags:
#A11yTCS; #A11ySev2; #A11yMAS; #DesktopWeb; #ChromiumEdge; #WCAG1.3.1; #InfoandRelationships; #BM_VisualStudioCodeServices_Web_July2024; #Visual Studio Code Services; #Benchmark; #Win11; #FTP; #Desktopweb.
## Environment Details: 
Application Name:  Visual Studio Code Services
URL: https://code.visualstudio.com/docs/?dv=win64user&dark-plus-v2
Microsoft Edge version 126.0.2592.68 (Official build) (64-bit)
 ## Repro steps:
1. Launch the application URL https://code.visualstudio.com/download?dark-plus-v2 and press ENTER.
2. Now Press TAB to "Docs" in the header section and observe that "Docs" link is not visible.

**Note: Similar issue is also observed with all the URLS**

1. https://code.visualstudio.com/docs/python/editing?dark-plus-v2
2. https://code.visualstudio.com/docs/python/jupyter-support-py?dark-plus-v2
3. https://code.visualstudio.com/docs/python/linting?dark-plus-v2
4. https://code.visualstudio.com/docs/remote/remote-overview?dark-plus-v2
5. https://code.visualstudio.com/docs/remote/troubleshooting?dark-plus-v2
6. https://code.visualstudio.com/docs/remote/wsl?dark-plus-v2

## Actual
"Docs" link is not visible in the "Visual studio code page"
## Expected
"Docs" link is clearly visible when focus lands on the "Docs" button
## User Impact
All users will get impacted if the "docs" link in not visually appearing in the "visual studio code" page.
## Attachments
![doc link](https://github.com/microsoft/vscode-docs/assets/151031203/c9f465e7-0328-4e7e-b873-e598c12b7271)

![doc2](https://github.com/microsoft/vscode-docs/assets/151031203/e10eb19b-1a05-495d-a589-bdabaf4e94d1)
![doc1](https://github.com/microsoft/vscode-docs/assets/151031203/a35a4957-69d1-404a-950e-d902112de6aa)

**Comment by daviddossett** [comment]
Hm, I'm seeing the focus border here. 

![Image](https://github.com/microsoft/vscode-docs/assets/25163139/d2a106eb-91a3-4047-a1f7-660cc9335fd0)

**Comment by PrasannaPolisetty123** [comment]
@daviddossett 
![image](https://github.com/microsoft/vscode-docs/assets/151031203/6f9a811e-4730-40cf-ae72-8f37393306e6)

"docs" link name is not visually present ,  the focus is going to the link, but visually "docs" link name is not visually appearing

**Comment by daviddossett** [comment]
Yeah I see it. Wondering if you're seeing old bits or something. I remember fixing this a couple weeks ago. Can't repro myself.

**Comment by daviddossett** [comment]
Ah, finally found the issue and fix

**Comment by 12-shweta** [comment]
@daviddossett  we don't have access to PR, could you please help us with environment link

**Comment by 12-shweta** [comment]
#Rev:shwarh

**Comment by 12-shweta** [comment]
#A11yPreview;

**Comment by SwatiJo** [comment]
@PrasannaPolisetty123  This issue has been fixed. Please close the bug. 
![Video-6-27-at-12 46](https://github.com/microsoft/vscode-docs/assets/167469595/e2781fcf-9d3a-4d58-8885-1ae7adbf95a2)

**Comment by SwatiJo** [comment]
@PrasannaPolisetty123  This issue has been fixed. Please close. 
![Recording-20240627_134757](https://github.com/microsoft/vscode-docs/assets/167469595/12c27265-36b5-409d-8356-420205d458f3)

**Comment by PrasannaPolisetty123** [comment]
Issue is fixed, hence closing the bug 
![image](https://github.com/microsoft/vscode-docs/assets/151031203/4d9abdc7-560b-42c9-baf2-2d596b0b9f2e)
